bucket/libreoffice.json

```diff
@@ -1,62 +1,69 @@
 {
-    "version": "7.3.2",
-    "description": "Powerful office suite",
+    "version": "7.3.4",
+    "description": "Powerful and free office suite, a successor to OpenOffice(.org).",
     "homepage": "https://libreoffice.org/",
     "license": "MPL-2.0",
     "suggest": {
-        "vcredist": "extras/vcredist2022"
+        "Visual C++ Redistributable 2015-2022": "extras/vcredist2022"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://download.documentfoundation.org/libreoffice/stable/7.3.4/win/x86_64/LibreOffice_7.3.4_Win_x64.msi",
+            "hash": "509c70c1c8136805480146b55e4bad5dc73b11ee47b4682b43cf07670109e176"
+        },
+        "32bit": {
+            "url": "https://download.documentfoundation.org/libreoffice/stable/7.3.4/win/x86/LibreOffice_7.3.4_Win_x86.msi",
+            "hash": "e73e2fd3d95f305a8517abe65c940e2b7de13d4271fb789f38f89c28df57933a"
+        }
     },
-    "url": "https://download.documentfoundation.org/libreoffice/portable/7.3.2/LibreOfficePortable_7.3.2_MultilingualStandard.paf.exe#/dl.7z",
-    "hash": "ae81da2a2bfb27a0cbdeffa65f712f8483a4f20049cbd1e02de2a36e6c3ff9d5",
     "pre_install": [
-        "if (!(Test-Path \"$persist_dir\\Data\\settings\")) {",
-        "    New-Item \"$dir\\Data\\settings\\LibreOfficePortableSettings.ini\" -Value \"[LibreOfficePortableSettings]`nInvalidPackageWarningShown=$version.0\" -Force | Out-Null",
-        "    if (Test-Path \"$env:AppData\\LibreOffice\") {",
-        "        info \"Copying old '$env:AppData\\LibreOffice' to '$persist_dir\\Data\\settings'\"",
-        "        Get-Item \"$env:AppData\\LibreOffice\\*\\*\" | Copy-Item -Destination \"$dir\\Data\\settings\" -Recurse -Force",
-        "    } else {",
-        "        Copy-Item \"$dir\\App\\DefaultData\\*\" \"$dir\\Data\" -Recurse -Force",
-        "    }",
```

Comment:
Should we migrate persisted data back to `$env:AppData\LibreOffice` and delete from persist?

Reply:
I prefer not.

If we are doing it, we should only copy from persist if `$env:AppData\LibreOffice` is **empty**.
Otherwise this will happen:

(1) User installs version `7.3.2`:  Config copied from `$persist_dir` -> `$Env:AppData\LibreOffice`.
(2) User updates to version `7.3.4`: Config copied again from `$persist_dir` -> `$Env:AppData\LibreOffice`. Overwriting existing config that are set by the user during the use of version `7.3.2`.

Reply:
I agree. We could also leave a note to the user but only if there is data in `$persist_dir` (i.e. upgrading from portable version)